.github/workflows/build.yml

```diff
@@ -76,3 +61,77 @@ jobs:
             opendj-doc-generated-ref/target/*.zip
             opendj-dsml-servlet/target/*.war
             opendj-rest2ldap-servlet/target/*.war
+  build-docker:
```

Comment:
This one builds the Docker images in parallel and decoupled from the Maven build. I suppose this is ok, but I'm open for feedback.